lib/Common/Core/ConfigParser.cpp

```diff
@@ -513,7 +519,10 @@ HRESULT ConfigParser::SetOutputFile(const WCHAR* outputFile, const WCHAR* openMo
 
     char16 fileName[_MAX_PATH];
     char16 moduleName[_MAX_PATH];
-    GetModuleFileName(0, moduleName, _MAX_PATH);
+    if (!GetModuleFileName(0, moduleName, _MAX_PATH))
+    {
+        AssertMsg(FALSE, "Get filename of the current executable failed");
```

Comment:
Is it okay to continue execution if `GetModuleFileName` fails in release builds (where Asserts don't hit)?